### PR TITLE
Updating all the doc examples related to encryption/decryption stuff

### PR DESCRIPTION
**Pull request by Aaryan-R-S**
# Description

Updating all the doc examples related to encryption/decryption stuff.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Introducing new feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

Link to the relevant updated sections in the Netlify preview

- [ ] Link 1: [description of change]
- [ ] Link 2: [...]

# Checklist:

General
- [ ] I have performed a self-review of my code
- [ ] I have fixed all grammar issues (can use an AI tool to check), and explanations are in active voice
- [ ] I have checked the additions are concise
- [ ] Language is consistent with existing documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published (ie. SDK changes, node dependencies)

If I have added a new concept, I have
- [ ] included a beginner friendly explanation
- [ ] included a basic technical introduction and code sample
- [ ] new terms are defined, both in relevant new pages and in the glossary

### CODE REVIEW

File: docs/integrations/storage/irys.md [hunk 1]
```diff
@@ -60,61 +60,113 @@ dotenv.config();
...
+  LitAbility,
+  createSiweMessageWithRecaps,
+  generateAuthSig,
+} from "@lit-protocol/auth-helpers";
+
+async getSessionSignatures(){
+    // Connect to the wallet
+    const ethWallet = new ethers.Wallet(
+      "<your private key>"
+    );
```
Comment (joshLong145) [inline docs/integrations/storage/irys.md:118]: why are we creating the wallet in scope of the session generation? This is a bit strange as the wallet should be outside of this scope and instead and `authSig` should be signed in the `authNeededCallback` so perhaps we pass the wallet to it?

File: docs/network/migration-guide.md [hunk 2]
```diff
@@ -119,12 +127,70 @@ class LitV3 {  // or Class LitV4
     this.litNodeClient = client;
   }
 
+  async getSessionSignatures(){
+      // Connect to the wallet
+      const ethWallet = new ethers.Wallet(
+        "<your private key>"
+      );
```
Comment (joshLong145) [inline docs/network/migration-guide.md:134]: same as above

File: docs/integrations/storage/irys.md (hunk 1 above)
Comment (spacesailor24) [reply docs/integrations/storage/irys.md]: I've opted to just remove the code example from this doc and link to the `Get Session Sig` page - This helps us de-dup the code

File: docs/network/migration-guide.md (hunk 2 above)
Comment (spacesailor24) [reply docs/network/migration-guide.md]: [Moved it](https://github.com/LIT-Protocol/docs/pull/279/commits/4f4aa2b5c0121389d4e8efabdab4791ae28ffd78) into the `constructor`

File: docs/sdk/access-control/encryption.md [hunk 3]
```diff
@@ -97,124 +197,374 @@ const accessControlConditions = [
...
-#### Passing an AuthSig
+#### Encryption
 
-First, obtain an authSig. This will ask MetaMask to sign a message proving the holder owns the crypto address. The chain we are using in this example is `ethereum`, you can check out additional supported chains [here](../../resources/supported-chains.md).
+To encrypt a string, use one of the following functions:
 
-```js
-const authSig = await LitJsSdk.checkAndSignAuthMessage({ chain: "ethereum" });
-```
+- [encryptString()](https://v3.api-docs.getlit.dev/functions/encryption_src.encryptString.html) - Used to encrypt the raw string.
```
Comment (joshLong145) [inline docs/sdk/access-control/encryption.md:204]: can we use local pathing instead of the url?

File: docs/sdk/access-control/encryption.md [hunk 4]
```diff
@@ -97,124 +197,374 @@ const accessControlConditions = [
...
+- [zipAndEncryptString()](https://v3.api-docs.getlit.dev/functions/encryption_src.zipAndEncryptString.html) - Compresses the string (using [JSZip](https://www.npmjs.com/package/jszip)) before encrypting it. This is useful for saving space, but takes additional time to perform the zip.
```
Comment (joshLong145) [inline docs/sdk/access-control/encryption.md:205]: same as above

File: docs/sdk/access-control/encryption.md [hunk 5]
```diff
@@ -97,124 +197,374 @@ const accessControlConditions = [
...
-- If you are encrypting a small file (less than 20mb) then you can use `encryptFileAndZipWithMetadata()` which will zip the file, and include all metadata in the zip, so you don't have to store anything else. If you want to store the metadata yourself, manually, you can use `zipAndEncryptFiles()` instead.
+- [encryptFile()](https://v3.api-docs.getlit.dev/functions/encryption_src.encryptFile.html) - Used to encrypt a file without doing any zipping or packing. Because zipping larger files takes time, this function is useful when encrypting large files ( > 20mb). This also requires that you store the file metadata.
+- [encryptFileAndZipWithMetadata()](https://v3.api-docs.getlit.dev/functions/encryption_src.encryptFileAndZipWithMetadata.html) - Used to encrypt a file and then zip it up with the metadata (using [JSZip](https://www.npmjs.com/package/jszip)). This is useful for when you don't want to store the file metadata separately.
+- [zipAndEncryptFiles()](https://v3.api-docs.getlit.dev/functions/encryption_src.zipAndEncryptFiles.html) - Used to zip and encrypt multiple files. This does **not** include the file metadatas in the zip, so you must store those yourself.
```
Comment (joshLong145) [inline docs/sdk/access-control/encryption.md:211]: Same as above, local pathing over `url`

File: docs/sdk/access-control/encryption.md [hunk 6]
```diff
@@ -97,124 +197,374 @@ const accessControlConditions = [
...
-  {
-    accessControlConditions,
-    authSig,
-    chain: "ethereum",
-    dataToEncrypt: "this is a secret message",
-  },
-  litNodeClient
-);
-```
+- [encryptToJson()](https://v3.api-docs.getlit.dev/functions/encryption_src.encryptToJson.html) - Used to encrypt a string or file and serialise all the metadata required to decrypt i.e. accessControlConditions, evmContractConditions, solRpcConditions, unifiedAccessControlConditions & chain to JSON. It is useful for encrypting/decrypting data in IPFS or other storage without compressing it in a ZIP file.
```
Comment (joshLong145) [inline docs/sdk/access-control/encryption.md:215]: same as above

File: docs/sdk/access-control/encryption.md [hunk 7]
```diff
@@ -97,124 +197,374 @@ const accessControlConditions = [
...
-    accessControlConditions,
-    ciphertext,
-    dataToEncryptHash,
-    authSig,
-    chain: "ethereum",
-  },
-  litNodeClient
-);
+Check out a complete example [here](https://github.com/LIT-Protocol/js-sdk/blob/1286138adc09ac2d34371f3ac12a9088ada367ec/e2e-nodejs/group-rli/test-rli-from-lit-node-client-diff-delegatee.mjs).
+
```
Comment (joshLong145) [inline docs/sdk/access-control/encryption.md:341]: We are removing the `e2e` nodejs tests soon. so perhaps we just remove this link and update once we merge in `auth unification` with the new `local-test` test cases?

**Review by spacesailor24 — APPROVED** [review]

**Pull request merged**